### PR TITLE
feat: make summarizer the default profile

### PR DESCRIPTION
**Pull request by lukealvoeiro**

### CODE REVIEW

File: src/goose/profile.py [hunk 1]
```diff
@@ -49,6 +49,6 @@ def default_profile(provider: str, processor: str, accelerator: str, **kwargs: D
         provider=provider,
         processor=processor,
         accelerator=accelerator,
-        moderator="truncate",
+        moderator="summarize",
```
Comment (zakiali) [inline src/goose/profile.py:52]: We need to get the token offset / fraction of token count into the Moderator to enable this by default. Currently, it'll summarize the entire context and replace, which I don't think we want.

Comment (lukealvoeiro) [reply src/goose/profile.py:52]: Yup!

**Review by michaelneale — CHANGES_REQUESTED** [review]
I don't think we should do this until summarizer is updated to not hard code gpt4o https://github.com/square/exchange/issues/57

**Comment by michaelneale** [comment]
closing this as it is more likely to be synopsis

**Pull request closed without merging**